sdk/identity/azure-identity/src/main/java/com/azure/identity/AppServiceMsiCredential.java

```diff
@@ -29,10 +29,10 @@
     AppServiceMsiCredential(String clientId, IdentityClient identityClient) {
         super(clientId, identityClient, "AZURE APP SERVICE MSI/IDENTITY ENDPOINT");
         Configuration configuration = Configuration.getGlobalConfiguration().clone();
-        this.msiEndpoint = configuration.get(Configuration.PROPERTY_MSI_ENDPOINT);
-        this.msiSecret = configuration.get(Configuration.PROPERTY_MSI_SECRET);
-        if (msiEndpoint != null) {
-            validateEndpointProtocol(this.msiEndpoint, "MSI", logger);
+        this.identityEndpoint = configuration.get(Configuration.PROPERTY_IDENTITY_ENDPOINT);
+        this.identityHeader = configuration.get(Configuration.PROPERTY_IDENTITY_HEADER);
```

Comment:
So, has this been using the wrong environment properties or was this just a renaming of the static variable being used?